src/hooks/vaultsHooks.ts

```diff
@@ -67,10 +68,18 @@ export const useLoadedVaultDetails = () => {
   const vaultDetailsResult = useQuery({
     queryKey: ['vaultDetails'],
     queryFn: async () => {
-      const callResult = await getMegavaultHistoricalPnl();
+      const toTyped = (res: any) =>
+        VaultCalculator.getVaultHistoricalPnlResponse(safeStringifyForAbacusParsing(res));
+      const [dailyResult, hourlyResult] = await Promise.all([
+        getMegavaultHistoricalPnl().then(toTyped),
```

Comment:
maybe just pass in the PnlTickInterval.day here to be explicit?

also why is it `PnlTickInterval.day` but `PnlTickInterval.HOUR` is all caps lmao

Reply:
lol I was wondering that too